onboarding.md

```diff
@@ -0,0 +1,28 @@
+# Onboarding
+
+All participants in the OpenJS Foundation projects and groups must follow the [Code of Conduct](https://github.com/openjs-foundation/cross-project-council/blob/master/CODE_OF_CONDUCT.md).
+There are further expectations for members who represent the Standards Working Group (hereby called representatives).
+
+It is understood that representatives will be a part of individual or their company specific projects. But while representing
+the Standards Working Group, they should take decisions/discuss/promote in the interest of Standards Group. 
```

Comment:
> interest of the Standards Group

Does the standards group have interests separate from those of foundation-supported projects?

Reply:
> Does the standards group have interests separate from those of foundation-supported projects?

I don't think there will be. But I am not 100% sure at this point. 

But the intention here is that the representative should have the group's interest rather than their project's interest if they are also representing the project.